products/compute/api.yaml

```
@@ -8894,6 +8965,205 @@ objects:
+        wait_ms: 1000
+      result: !ruby/object:Api::Async::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::Async::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::Async::Error
+        path: 'error/errors'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'region'
+        resource: 'Region'
+        imports: 'name'
+        input: true
+        required: true
+        description: |
+          The region where the regional proxy resides.
+    properties:
+      - !ruby/object:Api::Type::Time
+        name: 'creationTimestamp'
+        description: 'Creation timestamp in RFC3339 text format.'
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: 'An optional description of this resource.'
+        input: true
+      - !ruby/object:Api::Type::Integer
+        name: 'id'
+        description: 'The unique identifier for the resource.'
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          Name of the resource. Provided by the client when the resource is
+          created. The name must be 1-63 characters long, and comply with
+          RFC1035. Specifically, the name must be 1-63 characters long and match
+          the regular expression `[a-z]([-a-z0-9]*[a-z0-9])?` which means the
+          first character must be a lowercase letter, and all following
+          characters must be a dash, lowercase letter, or digit, except the last
+          character, which cannot be a dash.
+        input: true
+        required: true
+     # This field is present in the schema but as of 2019 Sep 23 attempting to set it fails with
+     # a 400 "QUIC override is supported only with global TargetHttpsProxy". jamessynge@ said in an
+     # email sent on 2019 Sep 20 that support for this "is probably far in the future."
+     #- !ruby/object:Api::Type::Enum
+     #  name: 'quicOverride'
+     #  description: |
+     #    Specifies the QUIC override policy for this resource. This determines
+     #    whether the load balancer will attempt to negotiate QUIC with clients
+     #    or not. Can specify one of NONE, ENABLE, or DISABLE. If NONE is
+     #    specified, uses the QUIC policy with no user overrides, which is
+     #    equivalent to DISABLE. Not specifying this field is equivalent to
+     #    specifying NONE.
+     #  values:
+     #    - :NONE
+     #    - :ENABLE
+     #    - :DISABLE
+     #  update_verb: :POST
+     #  update_url:
+     #    'projects/{{project}}/regions/{{region}}/targetHttpsProxies/{{name}}/setQuicOverride'
+      - !ruby/object:Api::Type::Array
+        name: 'sslCertificates'
+        description: |
+          A list of SslCertificate resources that are used to authenticate
```

I can definitely see why they did this in the docs (probably since it's the same object under the hood), but given that our products have a notion of resource I think it makes for a better UX if we call it out explicitly.
```suggestion
          A list of RegionSslCertificate resources that are used to authenticate
```